tensorflow/training/docker/2.8/py3/Dockerfile.cpu

```diff
@@ -36,7 +27,11 @@ ARG PYTHON_VERSION=3.9.10
 ARG PYTHON_PIP=python3-pip
 ARG PIP=pip3
```

Comment:
What's the difference between `PYTHON_PIP` and `pip`? Do we need both?

Reply:
right it's unclear what PYTHON_PIP is doing, I removed it